assets/js/components/events/EventsDashboard.jsx

```diff
@@ -359,7 +359,10 @@ class EventsDashboard extends Component {
         render: (data, row) => {
           const integrationResponses = row.integrations && row.integrations.filter(i => i.subCategory === 'uplink_integration_res');
           const integrationError = integrationResponses.findIndex(i => i.status === 'error') === -1 ? false : true;
-          const integrationMissing = row.integrations.findIndex(i => i.id === 'no_channel') === -1 ? false : true;
+
+          const externalIntegrations = row.integrations.filter(i => i.name !== 'Internal Integration');
+          const integrationMissing = externalIntegrations.findIndex(i => i.id === 'no_channel') === -1 ? false : true;
```

Comment:
might need to add a case where if externalIntegrations length is 0, it should be true

Reply:
true! addressed.